idd/Energy+.idd.in

```diff
@@ -70729,12 +70729,12 @@ ChillerHeater:Absorption:DoubleEffect,
        \minimum> 0.0
        \default 1.0
 
-HeatPump:WaterToWater:EIR:Cooling,
+HeatPump:PlantLoop:EIR:Cooling,
         \memo An EIR formulated water to water heat pump model, cooling operation.
-        \min-fields 15
+        \min-fields 16
```

Comment:
@mitchute This should be 15, 'cause 15 was wrong before. But #7659 will move this to 14, so ultimately before this merges and after #7659 is in, we need to make sure this is 15. Same for the Heating object.

Reply:
@mjwitte noted. I'll make a reminder for myself to keep an eye on this.